### DISCUSSION TITLE
新版本 播放本地音乐加载不出歌词

### DISCUSSION THREAD

**Discussion by xiele88787** [amll-dev/applemusic-like-lyrics#233] [General]
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/98011299/509b5e12-1e1d-43c9-818f-4c408b3721be)
加载不出错误码

**Comment by Linho1219** [comment]
AMLL bncm 插件已停止维护，最终版本存档于 <https://github.com/amll-dev/applemusic-like-lyrics-proto>。

现关闭此讨论，敬请谅解。
